.gitlab/kernel_matrix_testing/common.yml

```diff
@@ -200,7 +200,7 @@
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner$TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX:$TEST_INFRA_DEFINITIONS_BUILDIMAGES
   tags: ["arch:amd64"]
   before_script:
-    - GITLAB_TOKEN=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $GITLAB_SCHEDULER_TOKEN) || exit $?; export GITLAB_TOKEN
+    - GITLAB_TOKEN=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $GITLAB_TOKEN write_api) || exit $?; export GITLAB_TOKEN
```

Comment:
I don't think we need the `write_api` token here, `read_api` should suffice. We are only reading from the Gitlab API.

Reply:
You are perfectly right, that was my initial intention and I mixed things up, sorry.

Reply:
Nice catch, as it was a `cleanup` I wrongly assumed you were doing write operations on gitlab.
Thanks!